### ISSUE TITLE
UI/UX updates on admin app Users page

### ISSUE THREAD

**Issue by amandacilek** [medic/medic#5400]
Assumes text updates in issue #5386 

**Main page**
- [ ] For “Place” and “Contact” columns, display the actual names of the places/contacts instead of just a checkmark. 
- [ ] It is not intuitive that clicking on a row will allow you to edit. On hover, display an “Edit” button on the right side of the row where “Delete” is now. Keep the whole row as the click target.
- [ ] Move the delete button from the main page to the edit user pop up window. 

Like this:
![admin-users](https://user-images.githubusercontent.com/26235957/52812598-af683280-304c-11e9-9bc5-ab316b36ee72.jpg)

**Add/edit user pop up window**
- [ ] When setting place and associated contact, the inset layout and numbering before the names in the dropdown is confusing e.g. “1. XYZ,  2. ABC”. Can this just use our familiar breadcrumbs pattern instead?
- [ ] When setting the associated contact, the contacts that appear in the list should belong to (be restricted to) the place that you’ve already selected.

**Comment by SCdF** [comment]
Apologies if this is dealt with in another ticket, but we should also paginate this page. If you go onto a partner that has a reasonable number of users this page is really hard to use and takes a very long time to load.

We could even consider having a filter search bar at the top where you can type a name and have it live-return filtered results back. Due to CouchDB being CouchDB it's hard to do much more than filtering by username, full name and maybe phone number (I forget where that is stored), but it's better than nothing.

**Comment by dianabarsan** [comment]
@SCdF depends on which phone number you'd wanna filter by. 
There is a phone field in the user form which is saved in the `user-settings` doc, but there is also a phone field in the `contact` doc that is associated to the user. 

Since we're suggesting things, I particularly like alphabetical pagination :)

**Comment by amandacilek** [comment]
I agree about the pagination and have added that to the list above, thanks Stefan. I suppose let's save the filter/search functionality for now. I have other pages I'd like similar functionality, and I've put some larger requests like that in a separate backlog.

**Comment by ssykx4** [comment]
Hi, may I work on this issue? Has it still not been resolved?

**Comment by dianabarsan** [comment]
Hi @ssykx4 . The Admin App is currently under reconstruction by another team, to migrate it to use Angular 20. This can only be addressed after the migration.

**Comment by kennsippell** [comment]
<img width="949" height="423" alt="Image" src="https://github.com/user-attachments/assets/de9cd904-1911-4054-a4c6-6948286092ba" />

Many complaints this week about personel in nairobi unable to load users page because request is too heavy. Would <3 pagination here!
